src/OrderFormAcceptanceTests.Steps/Steps/CommencementDate.cs

```diff
@@ -52,6 +52,38 @@ public void ThenTheUserIsAbleToManageTheCommencementDateSection()
         {
             Test.Pages.OrderForm.ClickEditCommencementDate();
             Test.Pages.OrderForm.EditNamedSectionPageDisplayed("commencement date").Should().BeTrue();
-        }        
+        }
+        
+        [Then(@"the enabled sections are Order description")]
+        public void ThenTheEnabledSectionsAreOrderDescription()
+        {
+            Test.Pages.OrderForm.OrderDescriptionEnabled().Should().BeTrue();
+        }
+        
+        [Then(@"Call Off Ordering Party")]
+        public void ThenCallOffOrderingParty()
+        {
+            Test.Pages.OrderForm.CallOffOrderingPartyEnabled().Should().BeTrue();
+        }
+        
+        [Then(@"Supplier")]
+        public void ThenSupplier()
+        {
+            Test.Pages.OrderForm.SupplierInformationEnabled().Should().BeTrue();
+        }
```

Comment:
I took these out to reuse existing steps

Reply:
I'm in process of removing these, will push a commit when local run passes